lib/src/screens/dashboard/widgets/balance_page.dart

```diff
@@ -8,7 +11,9 @@ class BalancePage extends StatelessWidget {
   BalancePage({@required this.dashboardViewModel});
 
   final DashboardViewModel dashboardViewModel;
-
+  ThemeBase get currentTheme => getIt.get<SettingsStore>().currentTheme;
+  Color get backgroundLightColor =>
+      currentTheme.type == ThemeType.bright ? Colors.transparent : Colors.white;
```

Comment:
Add SettingsStore as property to BalancePage class and pass SettingsStore instance through constructor and inject it in di.dart resolver

Reply:
Okay, Sir